server/src/main/scala/services/exports/flights/templates/BhxFlightsWithSplitsExportWithCombinedTerminals.scala

```diff
@@ -0,0 +1,21 @@
+package services.exports.flights.templates
+
+import actors.PartitionedPortStateActor.{FlightsRequest, GetFlightsForTerminals}
+import drt.shared.Terminals._
+import drt.shared._
+
+trait BhxFlightsWithSplitsExportWithCombinedTerminals {
+  val terminal: Terminal
+  val start: SDateLike
+  val end: SDateLike
+
+  val terminalsToQuery: Seq[Terminal] = Seq(T1, T2)
+
+  val flightsFilter: (ApiFlightWithSplits, Terminal) => Boolean = (fws, _) => terminalsToQuery.contains(fws.apiFlight.Terminal)
+
+  val requestForDiversions: FlightsRequest = GetFlightsForTerminals(start.millisSinceEpoch, end.millisSinceEpoch, terminalsToQuery)
+}
+
+case class BhxFlightsWithSplitsWithoutActualApiExportImpl(start: SDateLike, end: SDateLike, terminal: Terminal) extends FlightsWithSplitsWithoutActualApiExport with BhxFlightsWithSplitsExportWithCombinedTerminals
```

Comment:
You're right. I didn't mean to leave them with the 'Impl' suffix. I've removed it improved the naming now too since they didn't mention the fact they were to do with red lists